### ISSUE TITLE
Updates to FAQ page for CHS merger

### ISSUE THREAD

**Issue by mekline** [lookit/lookit-api#1179]
# TL;DR

 FAQ (https://lookit.mit.edu/faq/)
    `web/templates/web/faq.html`
   This page has lots of references to Lookit as the general site and/or platform, e.g.
   _"Here at **Lookit** we are very concerned about protecting your and your children's data"_
   _"**Lookit** supports recent versions of Chrome and Firefox."_
   _"If you are asking who runs this website, the answer is that we are a collaboration between two projects for online studies with children, **Lookit** and Children Helping Science. **Lookit** was founded by Kim Scott and Laura Schulz at MIT..."_ (...followed by more info about the history of Lookit and CHS. I don't think this section will be confusing for families but wanted to flag it for review).

# Narrative
What is the user story for this feature? the format should be as follows, e.g.:  "As a Researcher, I would like to have a new view to see all consented videos for a given experiment."

# Acceptance Criteria
A precise description of how/when this issue will be considered closed, e.g.:

- [ ] Tests for new feature
- [ ] QA on new feature
- [ ] Sign off from Kim

# Implementation Notes
Leave important notes/suggestions here for implementation (important files, suggested approaches, etc.)

- Add "complaint" FAQ item for parents if they ahve a problem study

**Comment by mekline** [comment]
I'll add updates to individual FAQ items in comments on their own, and will specify once I've added all the changes!  Changes are bolded

# What happens when my child participates in a research study?

If you have a child and would like to participate, **create an account (https://lookit.mit.edu/signup/)** and take a look at what we have available for your child's age range. Some studies can be done at any time, and some involve a scheduled video chat with a university-based researcher. **Many** studies require a laptop/desktop with a webcam. The study description will tell you if you need any other supplies (such as a marker and paper, or a high chair for your baby to sit in.)

*add the link to 'create an account', it just says [LINK] at the moment*

...(no changes to intervening paragraphs)...

Some portions of the study will be automatically recorded using your webcam and sent securely to **Children Helping Science.** Trained researchers will watch the video and record your child's responses--for instance, which way he pointed, or how long she looked at each image. We'll put these together with responses from lots of other children to learn more about how kids think!

**Comment by mekline** [comment]
# Who creates the studies?

...(no changes to earlier paragraphs)...

If a scientist wants to use this platform, their institution has to sign a contract with MIT where they agree to the [Terms of Use](https://lookit.mit.edu/termsofuse/) and certify that their studies will be reviewed and approved by an institutional review board. Studies are also subject to review and approval by **Children Helping Science**. As of **April 2023**, we have agreements with **over 90 universities**, including institutions in the United States, Canada, the United Kingdom, and European Union.

**Comment by mekline** [comment]
# How is our information kept secure and confidential?

Researchers using **Children Helping Science** agree to uphold a common set of standards about how data is protected and shared. For instance, they never publish children's names or birthdates, or information that could be used to calculate a birthdate.

**Our researcher interface** is designed with participant data protection as the top priority. For instance, a special **page** lets researchers confirm consent videos before they are able to download any other data from your session. Research groups can control who has access to what data in a very fine-grained way, for instance allowing an assistant to confirm consent and send gift cards, but not download study data.

All of your data, including video, is transmitted over a secure HTTPS connection to **Children Helping Science** storage, and is encrypted at rest. We take security very seriously; in addition to making sure any software we use is up-to-date, cloud servers are configured securely, and unit tests cover checking that accessing data requires correct permissions, we **conduct** a risk assessment and detailed manual penetration testing with a security contractor **every two years. Our last security assessment was conducted in December 2022.**

See also 'Who will see our video?'

**Comment by mekline** [comment]
# Can I learn the results of the research my child does?

...(no changes to earlier paragraphs)...

There can be a long lag between conducting a study and publication -- your five-year-olds might be eight-year-olds before the results are in press! So in addition to scientific publications, many of the labs that post studies on this website have ways for parents to sign up to receive updates. You can also set your communication preferences to be notified by **Children Helping Science** when we have results from studies you participated in.

**Comment by mekline** [comment]
# I love this. How do I share this site?

Become a parent ambassador! One of the biggest challenges in developmental research is reaching families like you. With more children, we can also answer more sophisticated questions -- including questions about individual differences, the ways many different factors can interact to affect outcomes. Families like yours can help us make our science more representative and more reliable.

If you like what you are doing, please share this website **(https://www.childrenhelpingscience.com)**, with our sincere thanks. Research on child development would be impossible without the support of parents like you.

**Comment by mekline** [comment]
# What browsers are supported? 

**Children Helping Science officially supports recent versions of Chrome and Firefox. In particular, the Lookit engine is not currently able to support Internet Explorer or Safari.**

**Comment by mekline** [comment]
# My study isn't working, what can I do?

If you are trying to participate in a study and having difficulties, please start by contacting the researchers who made that specific study. If you still need help, you can also reach the **Children Helping Science** team for help at [lookit-tech@mit.edu](mailto:lookit-tech@mit.edu).

**Comment by mekline** [comment]
# What security measures do you implement?

Here at **Children Helping Science** we are very concerned about protecting your and your children's data, and our software is designed to take advantage of up-to-date security measures. You can read a bit about how we do this in our [documentation](https://lookit.readthedocs.io/en/develop/community-irb-and-legal-information.html#sub-processors-and-information-about-gdpr-compliance-dpas), and an updated HECVAT is available by emailing [lookit@mit.edu](mailto:lookit@mit.edu).

**Comment by mekline** [comment]
# How can I list my study?

**Visit our [documentation](https://lookit.readthedocs.io/en/develop/researchers-start-here.html) for a complete guide to posting your studies on Children Helping Science!**

In order to list your study on this website, you first will need to have your institution sign a contract with MIT where they agree to the [Terms of Use](https://lookit.mit.edu/termsofuse) and certify that their studies will be reviewed and approved by an institutional review board. You will also need to edit your IRB protocol to include online testing, or submit a new protocol for your proposed online study.

As of **April 2023**, we have agreements with **over 90 universities**, including institutions in the United States, Canada, the United Kingdom, and European Union. Please email [lookit@mit.edu](mailto:lookit-tech@mit.edu) if you are not sure whether your institution already has an agreement, or if you need any help at all getting your paperwork in order.

While you are completing these steps, you can get started on **Children Helping Science** by creating a lab account, creating your first study, and getting it peer reviewed by our researcher community. A step-by-step guide to getting started is available [here](https://lookit.readthedocs.io/en/develop/researchers-start-here.html).

**Comment by mekline** [comment]
# Where can I discuss best practices for online research with other researchers? 

**Children Helping Science** has a [Slack community](https://docs.google.com/forms/d/e/1FAIpQLScI2h7G6aUSJb-I3fGHw2nB8HcuaomuNLiwta2CXhGGF2ZL-Q/viewform) for researchers to ask for help, conduct peer review, and discuss online research methods.

...(no other changes)...

**Comment by mekline** [comment]
This is a new item, which should be placed after "Will we be paid for our participation"

# I want to review the studies I have participated in, or I have a concern and want to contact the researchers

Each study on Children Helping Science is run by a specific researcher team. To find contact information for a specific study, visit your [Study History](https://lookit.mit.edu/studies/history/) page. On this page you can also see information about when you participated in a study, and for studies that run here on our platform, watch your child's video from the session! 

If you have a more general question, want to report something to the Children Helping Science team, or need help reaching the researcher team, please contact us at [lookit@mit.edu](lookit@mit.edu)!

**Comment by mekline** [comment]
...and that's it for the FAQ page!
